ivy/data_classes/container/experimental/elementwise.py

```diff
@@ -2975,3 +2975,96 @@ def digamma(
             map_sequences=map_sequences,
             out=out,
         )
+
+    @staticmethod
+    def static_sparsify_tensor(
+        x: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        card: int,
+        /,
+        *,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
```

Comment:
All of these arguments can be `ivy.Container` as well. Please update these type-hints.